index.d.ts

```diff
@@ -2,6 +2,32 @@ import {GlobbyOptions} from 'globby';
 import {Options as CpFileOptions} from 'cp-file';
 
 declare namespace cpy {
+	interface SourceFile {
+		/**
+		Resolved path to file.
+		@example `/tmp/foo.bar`
+		*/
+		readonly path: string,
+
```

Comment:
You need to document `relativePath`too.

Reply:
It's internal field, API users don't need to know about it and they shouldn't use it.

Reply:
Well, nvm. It's public now.